### DISCUSSION TITLE
Aging Mechanism

### DISCUSSION THREAD

**Discussion by Jadhav-Sourabh** [pybamm-team/PyBaMM#2176] [General]
Anyone can please share some reference jupyter notebooks related to aging mechanisms (capacity fading, increase in internal resistance, SEI  growth, lithium plating, etc.)

**Comment by brosaplanella** [comment]
You can find examples of PyBaMM's various features in the [example notebooks folder](https://github.com/pybamm-team/PyBaMM/tree/develop/examples/notebooks). In particular, under the [models subfolder](https://github.com/pybamm-team/PyBaMM/tree/develop/examples/notebooks/models) you will find notebooks about degradation.

**Comment by lappemic** [comment]
For those in search of the notebook on degradation, it has been moved to the documentation section. You can access it directly through this link: [Degradation Notebook in PyBaMM Documentation](https://github.com/pybamm-team/PyBaMM/blob/develop/docs/source/examples/notebooks/models/coupled-degradation.ipynb).
